src/main/java/hudson/plugins/jira/versionparameter/JiraVersionParameterDefinition.java

```diff
@@ -61,13 +57,13 @@ public ParameterValue createValue(CLICommand command, String value) throws IOExc
     }
 
     public List<JiraVersionParameterDefinition.Result> getVersions() throws IOException {
-        Job<?, ?> context = Stapler.getCurrentRequest().findAncestorObject(Job.class);
```

Comment:
why renaming this? it's confusing as it generate some noise for nothing. (i.e make me feeling what happen here a lot of changes!)